### PR TITLE
Add documentation for the inverse of relationship

### PR DESCRIPTION
**Pull request by meysammahfouzi**

### CODE REVIEW

**Comment by LukeTowers** [comment]
This is better suited under the usage example section, it doesn't need it's own section.

**Comment by meysammahfouzi** [comment]
@LukeTowers  Moved to the example section.

File: database-attachments.md [hunk 1]
```diff
@@ -149,3 +149,18 @@ Display the uploaded file on a page:
     }
 
     <img src="<?= $featuredImage ?>" alt="Featured Image">
+
+<a name="inverse-of-relation"></a>
+#### The inverse of the relation
```
Comment (LukeTowers) [inline database-attachments.md]: No need for this navigation tag or the header tag.

File: database-attachments.md [hunk 2]
```diff
@@ -149,3 +149,18 @@ Display the uploaded file on a page:
...
+
+    public $morphTo = [
+        'attachment' => []
+    ];
+    
+Example:  
+
+    $user = $file->attachment;
+    
+For more information read the [polymorphic relationships](https://octobercms.com/docs/database/relations#polymorphic-relations)
```
Comment (LukeTowers) [inline database-attachments.md]: Please use relative linking, not absolute. I.e. `[deferred binding](../database/relations#deferred-binding)`

**Pull request merged**